diff --git a/src/util.ts b/src/util.ts
index 5b431e5..f083708 100644
--- a/src/util.ts
+++ b/src/util.ts
@@ -10,10 +10,7 @@ let versionCache: string = null;
  * Get the version string of tslab from package.json.
  */
 export function getVersion(): string {
-  if (versionCache == null) {
-    versionCache = require("../package.json").version;
-  }
-  return versionCache;
+  return "@version@";
 }
 
 export function isValidModuleName(name: string): boolean {
